### PR TITLE
moving to the new branch

### PR DESCRIPTION
**Pull request by mohamedSabry0**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+			<article class="latest-posts">
+				<p>7 days ago</p>
+				<h2>this is article title</h2>
+				<h3>author</h3>
+				<span>Lorem ipsum dolor sit amet, an ignota saperet referrentur has, <br>
+					ad duo diam altera vocent. Latine feugait cu usu. Id pri feugait appellantur,<br>
+					 his iudico iudicabit ne. Vel an sumo ullum labore, ne scaevola consequuntur ius. <br>
+						Regione evertitur nam id. Dicit iudicabit nec et.</span>
+			</article>
+			<figure id="smash-subscribers">
```
Comment (pelzolga123) [inline index.html]: It should be an article

File: index.html [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+			<article class="latest-posts">
+				<p>7 days ago</p>
+				<h2>this is article title</h2>
+				<h3>author</h3>
+				<span>Lorem ipsum dolor sit amet, an ignota saperet referrentur has, <br>
+					ad duo diam altera vocent. Latine feugait cu usu. Id pri feugait appellantur,<br>
+					 his iudico iudicabit ne. Vel an sumo ullum labore, ne scaevola consequuntur ius. <br>
+						Regione evertitur nam id. Dicit iudicabit nec et.</span>
+			</article>
+			<figure class="smashing-tv">
```
Comment (pelzolga123) [inline index.html]: It should be an article

File: index.html [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+						<span>When was this published —</span>
+							Cu sit alia quando neglegentur. Et facilis comprehensam eum, viris mentitum<br>
+					 		id usu. His cu lorem fastidii qualisque. Sea consul eleifend suscipiantur ex, <br>
+					 		liberavisse comprehensam nec ne. Sea et expetenda quaerendum cotidieque.
+				</p>
+			</div>
+			</article>
+			<footer class="community-end"><p>More links on twitter</p></footer>			
+		</section>
+		<section class="book-ad-2">
```
Comment (pelzolga123) [inline index.html]: It should be an article

File: index.html [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+					<li><a href="#">CURATED POST TITLE</a></li>
+				</ul>
+			</div>
+			<figure>
```
Comment (pelzolga123) [inline index.html]: It should be an article.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project looks great but needs some changes in navbar and  main content.

**Comment by pelzolga123** [comment]
## Hi Team!  
#### Your project looks :+1: amazing! But there are a few things to change.

##### Navigation bar

* [ ] Project should use `div` and position elements as at the source page. That means, you need to look how it's done on original website and add all missing elements inside your navbar `li` elements (content is not important). The same with `div` element, it should contain `nav` like original website.  
[link to page](https://www.smashingmagazine.com/)

##### 1 article row

* [ ] It should contain 1 article, which takes full width. See comments above, I pointed, where you need to have articles.

* This part is a little messed up, please make `h1` and `button`  display properly. 

image

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

**Review by MauricioRobayo — APPROVED** [review]
Great job fixing the previous requirements 👍 , and very well done overall, the project looks great.

Keep up the good work 😄 .

**Status**: Approve. 
🚀

**Comment by RodolfoRodriguezMata** [comment]
Thanks for all your feedback! :)

**Pull request merged**